test/unittests/commands/excel2json/test_properties.py

```diff
@@ -222,6 +224,33 @@ def test_get_final_series_two_None(self) -> None:
         result = e2j._get_final_series(mandatory_check, no_attribute_check)
         self.assertEqual(result, expected)
 
+    def test_row2prop_problem(self) -> None:
+        test_series = pd.Series(
+            {
+                "name": "name_1",
+                "label_en": "label_en",
+                "label_de": "label_de",
+                "label_fr": "label_fr",
+                "label_it": "label_it",
+                "label_rm": "label_rm",
+                "comment_en": "comment_en",
+                "comment_de": "comment_de",
+                "comment_fr": "comment_fr",
+                "comment_it": "comment_it",
+                "comment_rm": "comment_rm",
+                "super": "super_1",
+                "subject": "subject_1",
+                "object": "object_1",
+                "gui_element": "Simple",
+                "gui_attributes": "max:1.4 / min:1.2",
+            }
+        )
+        res = e2j._row2prop(test_series, row_num=0)
+        assert isinstance(res, PropertyProblem)
+        assert res.prop_name == "name_1"
+        assert len(res.problem) == 1
+        assert isinstance(res.problem[0], InvalidExcelContentProblem)
```

Comment:
At first, I didn't understand why there is a problem. Could you add some more assertions, so that it becomes clear? For example:

```
        assert res.problem[0].actual_content == "max:1.4 / min:1.2"
        assert res.problem[0].expected_content == "attribute: value, attribute: value"
```